firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/internal/InstallActivity.java

```diff
@@ -143,6 +143,7 @@ private void startAndroidPackageInstallerIntent() {
     }
 
     intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+    intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK);
```

Comment:
I was debating on whether or not to put a comment. I don't feel like it's necessary, but am happy to hear other opinions.

Reply:
I think a comment could be nice here. Who knows who will be looking at this code a year from now :)

Reply:
Agreed :)